<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquipmentManageWindow</class>
 <widget class="QWidget" name="EquipmentManageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>728</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>575</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设备管理</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>311</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 2px solid gray;
	background-color: rgb(245, 245, 245);
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点1</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点2</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点3</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点4</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点7</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点10</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点13</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点5</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点8</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton11">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点11</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton14">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点14</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton6">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点6</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton9">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点9</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton12">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点12</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton15">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>节点15</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resource/Alarm/alarm(green).png</normaloff>Resource/Alarm/alarm(green).png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14px</string>
   </property>
   <property name="text">
    <string>区域1：总数15，在线13，离线1，异常1</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>181</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <item>
    <property name="text">
     <string>节点号：1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>状态：正常</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>设备类型：输入</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>软件版本：1.6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ip地址：192.168.12.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OSD信息：201楼电梯</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
